src/sidebar/components/test/group-list-section-test.js

```diff
@@ -53,17 +52,24 @@ describe('groupListSection', () => {
             { isScopedToUri: false, id: 1 },
           ];
 
-          const element = createGroupListSection(
+          const wrapper = createGroupListSection(
             fakeSectionGroups,
             fakeDisableOosGroupSelection
           );
 
-          fakeSectionGroups.forEach(g =>
-            assert.equal(
-              element.ctrl.isSelectable(g.id),
-              expectedIsSelectable[g.id]
+          const itemTypes = wrapper
```

Comment:
I find the logic here just a little tricky to follow quickly—possibly a comment would help?

Reply:
Added